source/server/admin/clusters_handler.h

```diff
@@ -13,6 +14,9 @@
 namespace Envoy {
 namespace Server {
 
+void setHealthFlag(Upstream::Host::HealthFlag flag, const Upstream::Host& host,
```

Comment:
Can you add a doxygen comment? It's a in the Envoy::Server space but it is realy just mutating an admin proto. At first glance I thought it might be mutating the health status in the host.